vault/data_source_aws_access_credentials.go

```diff
@@ -46,6 +46,14 @@ func awsAccessCredentialsDataSource() *schema.Resource {
 					return
 				},
 			},
+
+			"ttl": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "60m",
```

Comment:
I like that you set this default and I can see you got it from [here](https://github.com/hashicorp/vault/blob/master/builtin/logical/aws/path_user.go#L32). However, Vault actually has [a secret TTL hierarchy](https://www.vaultproject.io/docs/concepts/tokens.html) that will be overridden if an explicit TTL is passed in by the Vault provider.

Would you be willing to strip this default, and then below, only append the TTL when it's explicitly provided by the user?